android/app/src/main/java/org/uwaterloo/subletr/api/apis/DefaultApi.kt

```diff
@@ -272,6 +273,147 @@ class DefaultApi(basePath: kotlin.String = defaultBasePath, client: OkHttpClient
+                throw ClientException("Client error : ${localVarError.statusCode} ${localVarError.message.orEmpty()}", localVarError.statusCode, localVarResponse)
+            }
+            ResponseType.ServerError -> {
+                val localVarError = localVarResponse as ServerError<*>
+                throw ServerException("Server error : ${localVarError.statusCode} ${localVarError.message.orEmpty()}", localVarError.statusCode, localVarResponse)
+            }
+        }
+    }
+
+    /**
+     * 
+     * 
+     * @param requestBody 
+     * @return ApiResponse<ListingsImagesCreateResponse?>
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class)
+    suspend fun listingsImagesCreateWithHttpInfo(requestBody: kotlin.collections.List<kotlin.Int>) : ApiResponse<ListingsImagesCreateResponse?> = withContext(Dispatchers.IO) {
+        val localVariableConfig = listingsImagesCreateRequestConfig(requestBody = requestBody)
+
+        return@withContext request<kotlin.collections.List<kotlin.Int>, ListingsImagesCreateResponse>(
+            localVariableConfig
+        )
+    }
+
+    /**
+     * To obtain the request config of the operation listingsImagesCreate
+     *
+     * @param requestBody 
+     * @return RequestConfig
+     */
+    fun listingsImagesCreateRequestConfig(requestBody: kotlin.collections.List<kotlin.Int>) : RequestConfig<kotlin.collections.List<kotlin.Int>> {
+        val localVariableBody = requestBody
+        val localVariableQuery: MultiValueMap = mutableMapOf()
+        val localVariableHeaders: MutableMap<String, String> = mutableMapOf()
+        localVariableHeaders["Accept"] = "application/json"
+
+        return RequestConfig(
+            method = RequestMethod.POST,
+            path = "/listings/images/create",
+            query = localVariableQuery,
+            headers = localVariableHeaders,
+            requiresAuthentication = true,
+            body = localVariableBody
+        )
+    }
+
+    /**
+     * 
+     * 
+     * @param imageId 
+     * @return kotlin.collections.List<kotlin.Int>
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    suspend fun listingsImagesGet(imageId: kotlin.String) : kotlin.collections.List<kotlin.Int> = withContext(Dispatchers.IO) {
```

Comment:
Is the return value supposed to be type `kotlin.collections.List<kotlin.Int>`?

Reply:
Yep, but this won't work because of the string encoding. Fixed in https://github.com/andr-dev/CS446/pull/47.